package.json

```diff
@@ -1,35 +1,70 @@
 {
-  "name": "ts-template",
-  "version": "1.0.0",
-  "description": "Template repository with TypeScript support.",
-  "main": "build/index.ts",
+  "name": "ubiquibot-logger",
+  "version": "0.3.0",
+  "description": "Ubiquity logging module with Supabase support.",
   "author": "Ubiquity DAO",
   "license": "MIT",
-  "engines": {
-    "node": ">=20.10.0"
+  "type": "module",
+  "tsup": {
+    "entry": [
+      "src/supabase/helpers/tables/logs.ts",
+      "src/supabase/helpers/tables/pretty-logs.ts"
+    ],
+    "splitting": false,
+    "sourcemap": true,
+    "clean": true,
+    "dts": true,
+    "format": "esm"
+  },
+  "files": [
+    "dist/*"
+  ],
+  "main": "./dist/logs.js",
+  "typesVersions": {
+    "*": {
+      ".": [
+        "./dist/logs.d.ts"
+      ],
+      "pretty-logs": [
+        "./dist/pretty-logs.d.ts"
+      ]
+    }
+  },
+  "exports": {
+    ".": {
+      "types": "./dist/logs.d.ts",
+      "default": "./dist/logs.js"
+    },
+    "./pretty-logs": {
+      "types": "./dist/pretty-logs.d.ts",
+      "default": "./dist/pretty-logs.js"
+    }
   },
   "scripts": {
-    "start": "tsx build/esbuild-server.ts",
-    "build": "tsx build/esbuild-build.ts",
-    "format": "run-s format:prettier format:lint",
-    "format:lint": "eslint --fix .",
+    "build": "tsup",
+    "prepare": "husky install",
+    "format": "run-s format:*",
     "format:prettier": "prettier --write .",
-    "prepare": "husky install"
+    "format:eslint": "eslint . --ext .ts --cache --fix --plugin tsc --rule 'tsc/config: [2, { configFile: ./tsconfig.json }]'"
   },
   "keywords": [
     "typescript",
-    "template",
+    "logging",
+    "supabase",
     "dao",
     "ubiquity",
     "open-source"
   ],
   "dependencies": {
-    "dotenv": "^16.3.1"
+    "@supabase/supabase-js": "^2.4.0",
+    "dotenv": "^16.3.1",
+    "pino-std-serializers": "^6.2.2",
+    "probot": "^12.2.4"
   },
   "devDependencies": {
     "@commitlint/cli": "^18.4.3",
     "@commitlint/config-conventional": "^18.4.3",
-    "@types/node": "^20.10.0",
+    "@types/node": "^20.10.4",
```

Comment:
Why was this changed? Shouldn't we stay at LTS?

Reply:
Good catch, I think we can stay at ^20.10.0

Reply:
Okay you can address in a new pull request.

Reply:
@pavlovcik I did a test, changed to LTS `^20.10.0` and yarn installed 20.10.4 anyway, took the latest available patch version in `major.minor.patch` format:

+"@types/node@*", "@types/node@^20.10.0":
   version "20.10.4"
   resolved "https://registry.npmjs.org/@types/node/-/node-20.10.4.tgz"
   integrity sha512-D08YG6rr8X90YB56tSIuBaddy/UXAA9RKJoFvrsnogAum/0pmjkgi4+2nx9

Since node follows semantic versioning https://semver.org/ we can stay with `^20.10.4` as changing to `^20.10.0` will not have effect, the latest LTS patch version 20.10.x will be installed anyway.